modules/multi-architecture-creating-arm64-bootimage.adoc

```diff
@@ -5,9 +5,9 @@
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-creating-arm64-bootimage_{context}"]
 
-= Creating an ARM64 boot image using the Azure image gallery
+= Creating a 64-bit ARM boot image using the Azure image gallery
```

Comment:
I think this also applies for the case of creating an amd64 boot image when the cluster was installed with arm64 CP.

Reply:
This is resolved by duplicating the bootimage creation page and adding a rocedure for x86